pom.xml

```diff
@@ -140,37 +140,152 @@
+        
+        <version.jnpserver>5.0.3.GA</version.jnpserver>
+        <version.json>1.0</version.json>
+        <version.jta.api>1.0.0.Final</version.jta.api>
+        <version.junit>4.13.1</version.junit>
+        <version.karaf.services>4.2.1</version.karaf.services>
+        <version.lifecycle.mapping.plugin>1.0.0</version.lifecycle.mapping.plugin>
+        <version.maven-compiler-plugin>3.1</version.maven-compiler-plugin>
+        <version.microprofile.lra>1.0</version.microprofile.lra>
+        <version.narayana.tomcat.integration>1.0.0.Final</version.narayana.tomcat.integration>
+        <version.nimbus.jose.jwt>7.9</version.nimbus.jose.jwt>
+        <version.org.codehaus.jettison>1.3.2</version.org.codehaus.jettison>
+        <version.org.glassfish.javax.json>1.1</version.org.glassfish.javax.json>
+<!--         org.jboss -->
+        <version.org.jboss.arquillian>1.1.13.Final</version.org.jboss.arquillian>
+        <version.org.jboss.bom>1.0.0.Final</version.org.jboss.bom>
+        <version.org.jboss.ironjacamar>1.4.35.Final</version.org.jboss.ironjacamar>
+        <version.org.jboss.jboss-transaction-spi>7.5.2.Final</version.org.jboss.jboss-transaction-spi>
+        <version.org.jboss.logmanager-jboss-logmanager>1.4.0.Final</version.org.jboss.logmanager-jboss-logmanager>
+        <version.org.jboss.logmanager-log4j-jboss-logmanager>1.0.1.Final</version.org.jboss.logmanager-log4j-jboss-logmanager>
+        <version.org.jboss.naming>5.0.3.GA</version.org.jboss.naming>
+        <version.org.jboss.shrinkwrap.descriptors>2.0.0-alpha-5</version.org.jboss.shrinkwrap.descriptors>
+        <version.org.jboss.shrinkwrap>1.0.1</version.org.jboss.shrinkwrap>
+        <version.org.jboss.spec.javax.ejb>1.0.0.Final</version.org.jboss.spec.javax.ejb>
+        <version.org.jboss.spec.javax.jms>1.0.0.Final</version.org.jboss.spec.javax.jms>
+        <version.org.jboss.spec.javax.transaction>1.0.1.Final</version.org.jboss.spec.javax.transaction>
+        <version.org.jboss.stdio>1.0.1.GA</version.org.jboss.stdio>
+        <version.org.jboss.threads>2.3.2.Final</version.org.jboss.threads>
+        
+        <version.org.slf4j>1.7.2</version.org.slf4j>
+        <version.org.springframework.boot>2.0.9.RELEASE</version.org.springframework.boot>
+        <version.org.springframwork>5.2.22.RELEASE</version.org.springframwork>
+        <version.org.wildfly.arquillian>4.0.0.Alpha5</version.org.wildfly.arquillian>
+        <version.plugin.maven.dependency>2.3</version.plugin.maven.dependency>
+        <version.postgresql>9.1-901.jdbc4</version.postgresql>
+        <version.resteasy.api>3.0.5.Final</version.resteasy.api>
+        <version.resteasy.client>4.6.0.Final</version.resteasy.client>
+        <version.resteasy.jaxrs>4.0.0.Beta5</version.resteasy.jaxrs>
+        <version.resteasy>3.0.23.Final</version.resteasy>
+        <version.shrinkwrap>1.2.6</version.shrinkwrap>
+        <version.surefire.plugin>2.19.1</version.surefire.plugin>
```

Comment:
Could you please update this to the latest? In many quickstarts, surefire was already updated to 3.x

Reply:
Thanks @jmfinelli ! I think it is a good idea. I would go with the latest one then, which is the version 3.0.0-M7